### ISSUE TITLE
bibliographicResource can contain more than one ISBN

### ISSUE THREAD

**Issue by ltog** [linked-swissbib/vufind#13]
A bibliographicResource can contain more than one ISBN.

Example:

``` bash
curl -XGET 'sb-s6.swissbib.unibas.ch:8080/testsb/bibliographicResource/_search' -d '
{
"query": {
"filtered": {
"filter": {
"bool": {
  "must": {"term": {"@id": "http://data.swissbib.ch/resource/091861764"}}
}
}
}
}
}
'
```

This leads to an array-to-string conversion at https://github.com/linked-swissbib/vufind/blob/3ccdb5318fd52fa11c1f9fd9d1c3ddbfbc614894/module/LinkedSwissbib/src/LinkedSwissbib/RecordDriver/ElasticSearchRDF.php#L356

**Comment by ltog** [comment]
@PhilippKuntschik @marahellstern : What should we do in such a case? Just display any of the covers?

**Comment by marahellstern** [comment]
@ltog @PhilippKuntschik : Yes, just display any of the covers. I don't see the need to choose a particular cover and I wouldn’t know what criteria to use to make this choice/selection.
